index.bs

```diff
@@ -6596,6 +6631,14 @@ The following abstract operations are a grab-bag of utilities.
  1. Return ? [$StructuredDeserialize$](|serialized|, [=the current Realm=]).
 </div>
 
+<div algorithm>
+ <dfn abstract-op lt="StructuredTransferOrClone">StructuredTransferOrClone(|value|, |transferList|)</dfn>
+    performs the following steps:
+ 1. Let |serialized| be ! [$StructuredSerializeWithTransfer$](|value|, |transferList|).
```

Comment:
Both StructuredSerializeWithTransfer and StructuredDeserializeWithTransfer can throw, so they should have **?** in front of them, not **!**.

Reply:
Done